### ISSUE TITLE
Tile component for markdown pages

### ISSUE THREAD

**Issue by tobifroe** [unikubehq/docs#32]

**Comment by buschNT** [comment]
![Screenshot from 2021-12-02 10-37-26](https://user-images.githubusercontent.com/6874607/144396295-34a0bc4c-8960-4d6d-80ff-362d9ca95a00.png)

I guess it should be the same as in the Screenshot, but the link should go to a defined section of the page. @marty-2015 If you need something special, please let Tobi know.

PS: @tobifroe maybe we could vertically center the content in the Boxes (also in on  the home page)?

**Comment by marty-2015** [comment]
Yes, we can use the same style of boxes, thats all good with me. @Truthans do you agree?
